### DISCUSSION TITLE
Guess missing words

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#716] [Galaxy Trivia] [answered]
In the sentence below all of the missing words are anagrams of each other:

> When I pressed ==?== on the scales, it brought a ==?== to my eye, as a piece of metal stuck in my finger. I didn't ==?== this as a good experience.

**Comment by SANKARAMDAS** [comment, marked as answer]
When I pressed TARE on the scales, it brought a TEAR to my eye, as a piece of metal stuck in my finger. I didn't RATE this as a good experience.

**Comment by thuongtruong1009** [comment]
When I pressed SADNESS on the scales, it brought a SADNESS to my eye, as a piece of metal stuck in my finger. I didn't SENSED this as a good experience.

**Comment by peterbaccaro** [comment]
When I pressed **TARE** on the scales, it brought a **TEAR** to my eye, as a piece of metal stuck in my finger. I didn't **RATE** this as a good experience.
